Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM nvidia/cuda
+
+# Install miniconda
+RUN apt-get update && apt-get install -y wget
+RUN MINICONDA="Miniconda2-latest-Linux-x86_64.sh" && \
+    wget --quiet https://repo.continuum.io/miniconda/$MINICONDA && \
+    bash $MINICONDA -b -p /miniconda && \
+    rm -f $MINICONDA
+ENV PATH /miniconda/bin:$PATH
+
+# Add omnia
+RUN conda config --add channels omnia
+
+# Install deepchem
+RUN conda install --yes deepchem
```

Comment:
@jchodera I think we're moving towards an alpha-release right now. The major blockers are TF 1.0 upgrade and the shift to the MIT license. I like the idea of pinning to today's head in the docker image and then updating once the alpha is released.